test/matching-queue-plugin.js

```diff
@@ -173,6 +174,10 @@ tap.test('Matching Queue Plugin - should add the patient resource to the matchin
       }, (err, res) => {
         t.error(err)
 
+        // pull versionId from location header to verify that the ETag header is being set properly
+        const versionId = res.headers.location.replace(/\/fhir\/Patient\/.*\/_history\//, '')
+
+        t.equal(res.headers.etag, `W/"${versionId}"`)
```

Comment:
This seems like a random place to test that the etag is working. I seems like it should be its own test case to me.

Reply:
Tests moved in this [commit](https://github.com/jembi/hearth/pull/148/commits/be8a38a5ff3fd7dc7e05da12cc08241e80d84d7c)